.github/workflows/dockerimage.yml

```diff
@@ -37,9 +37,9 @@ jobs:
           --exclude 'polytracker/src/dfsan_rt/interception/*'
       - name: Python lint/typecheck 
         run: |
-          black --check polybuild polyprocess tests --line-length=127
-          mypy --ignore-missing-imports polybuild polyprocess tests
-      - name: Build the base image 
+          black --check polytracker tests --exclude '/(polytracker/src|polytracker/scripts)/' --line-length=127
```

Comment:
It's a bit of a hack. So, in Python, since we are now defining a `polytracker` package, it's standard practice to put that package in the `polytracker` subdirectory. But we already had our C++ code in there, which _also_ includes some _other_ Python scripts that are not included in the `polytracker` package (which we prevent by not including `__init__.py` files in their directories). Some of those scripts actually fail the `black` linting, which is why I just blanket exclude those directories. Note that `polybuild.py` is now included in the `scripts` directory, which is _excluded_ from this `black` check. So just keep in mind that `polybuild.py` will no longer be linted in CI. We should probably update all of the .py files in the `scripts` dir to satisfy `black` so we can remove that exclusion.